T-1/git-backup-helper/git-backup.sh

```
@@ -0,0 +1,85 @@
+    echo -e "${RED}錯誤：當前目錄不是 Git 倉庫${NC}"
+    echo -e "請在 Git 倉庫目錄中運行此命令，或者使用以下命令初始化一個新倉庫："
+    echo -e "  git init"
+    exit 1
+fi
+
+# 獲取當前時間戳
+TIMESTAMP=$(date +"%Y%m%d_%H%M%S")
+
+# 顯示當前倉庫信息
+CURRENT_BRANCH=$(git branch --show-current)
+REPO_NAME=$(basename -s .git $(git config --get remote.origin.url 2>/dev/null) || echo "local-repo")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)